vnext/src/Libraries/Components/ActivityIndicator/RCTActivityIndicatorViewNativeComponent.windows.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+const requireNativeComponent = require('../../ReactNative/requireNativeComponent');
+
+import type {ViewProps} from '../View/ViewPropTypes';
```

Comment:
>./View/ViewPropTypes [](start = 31, length = 20)

I guess this file doesn't exist. we only have ViewWindows.tsx in view folder

Reply:
We copy all the files from react-native into react-native-windows, so that file comes from react-native.